storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/StorageSharedBenchmarkingCli.java

```diff
@@ -103,23 +106,18 @@ private void runWorkload1() {
     StorageOptions retryStorageOptions =
         StorageOptions.newBuilder().setProjectId(project).setRetrySettings(retrySettings).build();
     Storage storageClient = retryStorageOptions.getService();
-    Path tempDir = Paths.get(System.getProperty("java.io.tmpdir"));
+    Path tempDir = Paths.get(tempDirLocation);
     ListeningExecutorService executorService =
         MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(workers));
     List<ApiFuture<String>> workloadRuns = new ArrayList<>();
     Range objectSizeRange = Range.of(objectSize);
     for (int i = 0; i < samples; i++) {
-      try {
-        TmpFile file =
-            DataGenerator.base64Characters()
-                .tempFile(tempDir, getRandomInt(objectSizeRange.min, objectSizeRange.max));
-        BlobInfo blob = BlobInfo.newBuilder(bucket, file.toString()).build();
-        workloadRuns.add(
-            convert(
-                executorService.submit(new Workload1(file, blob, storageClient, workers, api))));
-      } catch (IOException e) {
-        throw new RuntimeException(e);
-      }
+      int objectSize = getRandomInt(objectSizeRange.min, objectSizeRange.max);
+      PrintWriter pw = new PrintWriter(System.out);
```

Comment:
Probably want to set autoFlush to true
https://docs.oracle.com/javase/8/docs/api/java/io/PrintWriter.html#PrintWriter-java.io.OutputStream-boolean-